specs/protocol/abi.md

```diff
@@ -137,11 +378,11 @@ _Important note:_ For the JSON representation of receipts, we represent 64-bit u
 
 ```json
 {
-    "type":"Panic",
-    "id":"0x39150017c9e38e5e280432d546fae345d6ce6d8fe4710162c2e3a95a6faff051",
-    "reason":"1",
-    "pc":"0xffffffffffffffff",
-    "is":"0xfffffffffffffffe"
+   "type":"Panic",
```

Comment:
Why go from 4 to 3 indentation?

Reply:
So at first I thought `forc run --pretty-print` uses 3 spaces so I was running with 3 everywhere. Turns out that whatever JSON library we're using actually uses 2 spaces. Same goes for the JSON ABI file that the compiler generate. So, I'm now using 2 spaces everywhere.